how-to/editor-in-chief-guide.md

```diff
@@ -1,164 +1,297 @@
+everything at once.
 
-In some situations, the editor-in-chief initial checks may be passed down to an editor as follows:
+Before posting:
 
-* If the Editor in Chief is overwhelmed with package submissions to evaluate.
-* If the Editor in Chief simply is busy at the time and needs support with checks.
-* If the Editor in Chief thinks that the checks might be better served if done by an Editor
-(For instance, if a specific domain or technical expertise would support more effective checks).
+1. Review the editor-in-chief checklist.
+2. Estimate how long the checks might take and share that with the author.
+3. Perform all checks locally.
+4. Post the completed checklist and any feedback in a **single GitHub comment**.
+:::
+
+Copy the template below and add it to the issue.
+
+These checks confirm that the package meets the **minimum standards for
+review**. In many cases, going through this process improves the package itself.
+
+The EiC may ask another editor to complete the checks if:
+* They are overwhelmed with new submissions.
+* They are temporarily unavailable.
+* Another editor has relevant domain or technical expertise.
 
 (editor-checklist-template)=
 ### Editor-in-chief checklist
 
-Copy the template below to use it in the issue.
+Copy the template below and use it in the review issue.
 
 ```{include} ../appendices/editor-in-chief-checks.md
 ```
 
-### 3. ✔️ Ensure that the package onboarding survey is filled out.
-
-Thank the authors graciously for filling out our survey. They can
-skip sections of it if they wish. We do need their contact
-information to stay in touch about package maintenance. We also
-want to track their experience with our review process and
-organization.
-
-### 4. ✔️ Assign an editor to the issue to manage the rest of the review and tag the issue with the `1/editor-assigned` tag in GitHub
-
-Once the package initial checks are complete, and it is determined that
-the package is in scope for pyOpenSci review, the Editor in Chief will assign an
-editor to the review issue and add the `1/editor-assigned` tag in GitHub.
-This may involve finding a new (guest) editor
-as described in the [onboarding guide](onboarding-guide.md).
-If you as Editor in Chief do recruit a new editor,
-be sure to complete all the onboarding steps described
-[here](onboarding-a-new-editor) so that the new editor
-has everything they need to manage the review,
-such as GitHub permissions and access to the relevant channels
-in the pyOpenSci Slack team.
-The editor will now begin the process of finding reviewers
-for the package.
-[Check out the editor guide for more information on the process that an editor follows](editors-guide.md)
-
-```{admonition} Diversity in the editorial & reviewer  team is important
+### 4. ✔️ Ensure that the package onboarding survey is complete
+
+Use [this spreadsheet](https://docs.google.com/spreadsheets/d/1jEk-DDpkz14Z07OX_o1cN2vHzVbJO6mQ83ihGXsWkLc/edit?gid=930774086#gid=930774086)
```

Comment:
I removed emails and last names from this spreadsheet so it's a list of dates and github usernames which are all public already.